dataprovider/providers/cloud/data_provider.go

```diff
@@ -32,13 +32,25 @@ const (
 	cloudAccountNameField = "cloud.account.name"
 	cloudProviderField    = "cloud.provider"
 	cloudRegionField      = "cloud.region"
-	cloudProviderValue    = "aws"
+	cloudProjectIdField   = "cloud.project.id"
+	cloudProjectNameField = "cloud.project.name"
 )
 
+type Identity struct {
+	Provider     string
+	Account      string
+	AccountAlias string
+	ProjectId    string
+	ProjectName  string
+}
+
 type DataProvider struct {
-	log         *logp.Logger
-	accountId   string
-	accountName string
+	log          *logp.Logger
+	accountId    string
+	accountName  string
+	providerName string
+	projectName  string
+	projectId    string
 }
 
 func New(options ...Option) DataProvider {
```

Comment:
`cloud.New()` is not clear about which struct it initializes anymore

```suggestion
func NewDataProvider(options ...Option) DataProvider {
```